### PR TITLE
Update for GitKraken v8.3.3

### PR DESCRIPTION
**Pull request by Anselmoo**
It seems to be that a few new variables were added 

image

* Update `dracula-theme.jsonc`
* Add new screenshot

> If you're fixing a UI issue, make sure you take two screenshots. One that shows the actual bug and another that shows how you fixed it.

### CODE REVIEW

File: dracula-theme.jsonc [hunk 1]
```diff
@@ -6,35 +6,49 @@
...
       "red": "#ff5555",
       "orange": "#ffb86c",
       "yellow": "#f1fa8c",
       "green": "#50fa7b",
-      "purple": "#bd93f9",
-      "pink": "#ff79c6",
-      "cyan": "#8be9fd",
-      "app__bg0": "#282a36",
-      "comment": "#6272a4",
+      "blue": "#bd93f9",
```
Comment (Xtopher98) [inline dracula-theme.jsonc:18]: Why change purple to blue? The color in the Dracula documentation is called purple.

File: dracula-theme.jsonc [hunk 2]
```diff
@@ -6,35 +6,49 @@
...
+      "ltblue": "#8be9fd",
```
Comment (Xtopher98) [inline dracula-theme.jsonc:19]: Same with this. We should keep the same naming convention that Dracula does in [this list](https://draculatheme.com/contribute). This should stay cyan
```suggestion
      "cyan": "#8be9fd",
```

File: dracula-theme.jsonc [hunk 3]
```diff
@@ -6,35 +6,49 @@
...
       "input-bg-warn-color": "fade(@orange, 60%)",
-      "panel-border": "darken(@app__bg0, 4%)",
-      "section-border": "fade(#44475a, 8%)",
-      "subtle-border": "fade(#44475a, 4%)",
-      "modal-overlay-color": "rgb(40, 42, 54)",
+      "panel-border": "darken(@background, 4%)",
+      "section-border": "fade(@foreground, 8%)",
+      "subtle-border": "fade(@foreground, 4%)",
+      "modal-overlay-color": "@background",
+      // graph colors
```
Comment (Xtopher98) [inline dracula-theme.jsonc:33]: Why change the order of the graph colors? We should keep it consistent unless there is a good reason for the change. I do like the fade for the later colors.
```suggestion
      "graph-color-0": "@purple",
      "graph-color-1": "@cyan",
      "graph-color-2": "@green",
      "graph-color-3": "@orange",
      "graph-color-4": "@yellow",
      "graph-color-5": "fade(@purple, 8%)",
      "graph-color-6": "fade(@cyan, 8%)",
      "graph-color-7": "fade(@green, 8%)",
      "graph-color-8": "fade(@orange, 8%)",
      "graph-color-9": "fade(@yellow, 8%)",
```

File: dracula-theme.jsonc [hunk 4]
```diff
@@ -6,35 +6,49 @@
...
+      "app__bg0": "lighten(@background, 5%)",
       "toolbar__bg0": "lighten(saturate(@app__bg0, 3%), 6%)",
       "toolbar__bg1": "lighten(@toolbar__bg0, 4%)",
       "toolbar__bg2": "lighten(@toolbar__bg1, 6%)",
-      "panel__bg0": "lighten(@app__bg0, 5%)",
+      "panel__bg0": "lighten(@background, 5%)",
       "panel__bg1": "lighten(@panel__bg0, 5%)",
       "panel__bg2": "lighten(@panel__bg1, 5%)",
-      "input__bg": "fade(@app__bg0, 20%)",
+      "input__bg": "fade(@background, 10%)",
```
Comment (Xtopher98) [inline dracula-theme.jsonc:27]: Why the fade percent change? Unless there is a good reason we should keep it consistent.
```suggestion
      "input__bg": "fade(@background, 20%)",
```

File: dracula-theme.jsonc [hunk 5]
```diff
@@ -6,35 +6,49 @@
...
       // text colors
       // values starting with . aren't added to the CSS, they're just variables
-      ".text-color": "#f8f8f2",
+      ".text-color": "@foreground",
       "text-selected": "@.text-color",
       "text-normal": "fade(@.text-color, 75%)",
       "text-secondary": "fade(@.text-color, 60%)",
       "text-disabled": "fade(@.text-color, 40%)",
-      "text-accent": "@purple",
+      "text-accent": "@blue",
```
Comment (Xtopher98) [inline dracula-theme.jsonc:51]: Same as above
```suggestion
      "text-accent": "@purple",
```

File: dracula-theme.jsonc [hunk 6]
```diff
@@ -43,9 +57,9 @@
       "default-bg": "transparent",
       "default-hover": "transparent",
       "default-border-hover": "@text-selected",
-      "primary-border": "@purple",
-      "primary-bg": "fade(@purple, 20%)",
-      "primary-hover": "fade(@purple, 60%)",
+      "primary-border": "@blue",
+      "primary-bg": "fade(@blue, 20%)",
+      "primary-hover": "fade(@blue, 60%)",
```
Comment (Xtopher98) [inline dracula-theme.jsonc:62]: Same as above 
```suggestion
      "primary-border": "@purple",
      "primary-bg": "fade(@purple, 20%)",
      "primary-hover": "fade(@purple, 60%)",
```

File: dracula-theme.jsonc [hunk 7]
```diff
@@ -6,35 +6,49 @@
...
+      "graph-color-0": "@ltblue",
+      "graph-color-1": "@pink",
+      "graph-color-2": "@blue",
+      "graph-color-3": "@orange",
+      "graph-color-4": "@comment",
+      "graph-color-5": "fade(@ltblue, 8%)",
+      "graph-color-6": "fade(@pink, 8%)",
+      "graph-color-7": "fade(@blue, 8%)",
+      "graph-color-8": "fade(@orange, 8%)",
+      "graph-color-9": "fade(@comment, 8%)",
```
Comment (Xtopher98) [inline dracula-theme.jsonc:43]: Why change the order of the graph colors? We should keep it consistent unless there is a good reason for the change. I do like the fade for the later colors.
```suggestion
      "graph-color-0": "@purple",
      "graph-color-1": "@cyan",
      "graph-color-2": "@green",
      "graph-color-3": "@orange",
      "graph-color-4": "@yellow",
      "graph-color-5": "fade(@purple, 8%)",
      "graph-color-6": "fade(@cyan, 8%)",
      "graph-color-7": "fade(@green, 8%)",
      "graph-color-8": "fade(@orange, 8%)",
      "graph-color-9": "fade(@yellow, 8%)",
```

File: dracula-theme.jsonc [hunk 8]
```diff
@@ -56,53 +70,108 @@
...
+      "selected-row": "fade(@blue, 20%)",
       "warning-row": "fade(@orange, 60%)",
       "droppable": "fade(@yellow, 30%)",
       "drop-target": "fade(@green, 50%)",
-      "input--disabled": "fade(@comment, 10%)",
-      "link-color": "@cyan",
-      "form-control-focus": "@purple",
+      "input--disabled": "fade(@background, 10%)",
+      "link-color": "@ltblue",
+      "form-control-focus": "@blue",
```
Comment (Xtopher98) [inline dracula-theme.jsonc:81]:
```suggestion
      "link-color": "@cyan",
      "form-control-focus": "@purple",
```

File: dracula-theme.jsonc [hunk 9]
```diff
@@ -56,53 +70,108 @@
...
       // various app elements
-      "code-bg": "@app__bg0",
-      "scroll-thumb-border": "@app__bg0",
+      "scroll-thumb-border": "@background",
       "scroll-thumb-bg": "@toolbar__bg2",
       "scroll-thumb-bg-light": "lighten(@toolbar__bg2, 20%)",
-      "wip-status": "fade(@green, 40%)",
+      "wip-status": "fade(@blue, 20%)",
```
Comment (Xtopher98) [inline dracula-theme.jsonc:86]:
```suggestion
      "wip-status": "fade(@purple, 20%)",
```

File: dracula-theme.jsonc [hunk 10]
```diff
@@ -56,53 +70,108 @@
...
       "card__bg": "@panel__bg2",
-      "card-shadow": "rgb(40, 42, 54)",
+      "card-shadow": "fade(@background, 20%)",
+      "statusbar__warning-bg": "mixLess(@graph-color-7, @background, 50%)",
+      "label__yellow-color": "@yellow",
+      "label__light-blue-color": "@ltblue",
+      "label__purple-color": "@blue",
       // component states
-      "filtering": "fade(@purple, 50%)",
+      "filtering": "fade(@blue, 50%)",
```
Comment (Xtopher98) [inline dracula-theme.jsonc:94]:
```suggestion
      "filtering": "fade(@purple, 50%)",
```

File: dracula-theme.jsonc [hunk 11]
```diff
@@ -56,53 +70,108 @@
...
       "soloing": "fade(@orange, 50%)",
-      "checked-out": "fade(@purple, 30%)",
+      "checked-out": "fade(@green, 30%)",
```
Comment (Xtopher98) [inline dracula-theme.jsonc:96]: Why change the color?

**Review by Xtopher98 — CHANGES_REQUESTED** [review]
Good update overall adding the new values. 

We should keep the color names consistent with the [dracula documentation](https://draculatheme.com/contribute). I've added some suggestions to revert back.

I've added some other comments and suggestions in various places that I think are worth taking a look at. 

Let me know what you think.

**Comment by Anselmoo** [comment]
> Good update overall adding the new values.
> 
> We should keep the color names consistent with the [dracula documentation](https://draculatheme.com/contribute). I've added some suggestions to revert back.
> 
> I've added some other comments and suggestions in various places that I think are worth taking a look at.
> 
> Let me know what you think.

Thx a lot @Xtopher98, the suggestion makes sense and you checked the PR very carefully 👍

I will later make a new screen-shot and finish the PR

**Pull request merged**